css-scroll-anchoring-1/Overview.bs

```diff
@@ -229,6 +229,8 @@ These triggers are:
 	Note that this trigger applies regardless of whether the modified element is
 	on the path from the anchor node to the scrollable element.
 
+* The scroll offset of the scrollable element becoming zero.
```

Comment:
Should be `being` zero, right? Or at least that's closer to the chromium code I recall reading... So there's no change that needs to happen, just the scroll offset shouldn't be zero.

Reply:
Updated to "being" and fixed commit message.

nburris is working on a WPT test, will get that in soon..